docs/design/easydl-master.md

```diff
@@ -0,0 +1,234 @@
+# Training Master For DLRover
+The design describes how the architecture of the training master of DLRover.
+When the user submits an ElasticJob,
```

Comment:
It is better to briefly introduce the master's main role or function in dlrover at first. Especially, to specify its different roles with/without the brain.

Reply:
Done